### DISCUSSION TITLE
Welcome to person_following_bot Discussions!

### DISCUSSION THREAD

**Discussion by take2rohit** [IvLabs/person_following_bot#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nanda-672** [comment]
My name is Nanda Pramudia, a student from Bandung Institute of Technology in Indonesia. Currently I'm tasked to create follower bot myself. Is there a way to use this repo and test the tracker on a pc without the turtlebot?
